CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x](https://github.com/opensearch-project/k-NN/compare/2.15...2.x)
 ### Features
+* JNI layer interface to support Faiss binary index [#1778](https://github.com/opensearch-project/k-NN/pull/1778)
```

Comment:
eh this isnt a feature if it is not exposed to user. Just skip-change log for this and group it with binary field support feature.

Reply:
@navneet1v want to add it in change log

Reply:
This is a feature, but the description of the feature is not correct.

Reply:
Can we make it,
`Added the support for Binary Index in kNNEngine.`

Reply:
Updated. However, I agree with @jmazanec15 that it is not a feature that use can use though.

Reply:
Talked with @navneet1v offline. Will remove the change log entry for this PR.